src/testSchemas/enumSchema.ts

```diff
@@ -0,0 +1,105 @@
+import { JSONSchema } from "json-schema-to-ts"
+import { UISchema } from "../ui-schema"
+
+export const enumSizeSchema = {
+  type: "object",
+  properties: {
+    size: {
+      title: "Size",
+      type: "string",
+      enum: ["XS", "S", "M", "L", "XL"],
+      default: "M",
+    },
+  },
+  required: ["size"],
+} satisfies JSONSchema
+
+export const enumProfessionSchema = {
+  type: "object",
+  properties: {
+    profession: {
+      title: "Profession",
+      type: "string",
+      enum: [
+        "Footballer",
+        "Bob Ross Impersonator",
+        "Beat Boxer",
+        "Software Engineer",
+      ],
+    },
+  },
+  required: ["profession"],
+} satisfies JSONSchema
+
+export const enumPSISchema = {
+  type: "object",
+  properties: {
+    psi: {
+      title: "Approximate PSI",
+      type: "integer",
+      enum: [100, 1000, 10000],
+    },
+  },
+} satisfies JSONSchema
+
+export const enumSnakeCaseSchema = {
+  type: "object",
+  properties: {
+    humanReadableOptions: {
+      title: "Human Readable Options",
+      type: "string",
+      enum: ["option_1", "option_2", "option_3"],
+    },
+  },
+} satisfies JSONSchema
```

Comment:
prolly want `as const satisfies JSONSchema` here in order for UISchema type inference to work, no?

Reply:
Nope, it works as-is.

Reply:
that's cool! I wonder if there's some smart treatment of enum props by json-schema-to-ts then